### DISCUSSION TITLE
How to load data use relative path in quarto?

### DISCUSSION THREAD

**Discussion by lizhiwei1994** [quarto-dev/quarto-cli#2212] [Q&A]
Hi everyone.
In my slides, I need load some data. I can only use absolute paths to read in data, because when I use relative paths it reports an error.

For example, my absolute paths for `data1.csv` is: `D:/How to plot in R/examples/data1.csv`.

I want to use relative paths in my R project, such as `examples/data1.csv`.

In R markdown, we can choose the Knit Directory (pls see [How to correctly specify relative path](https://community.rstudio.com/t/how-to-correctly-specify-relative-path/9612/3)). But I don't find a similar button in quarto.

My folder structure is as follows.

How to plot in R/
├─ How to plot in R.Rproj
├─ slides/
│  ├─ index.qmd
├─ examples/
│  ├─ data1.csv/
│  ├─ data2.csv
│  ├─ data3.csv

**Comment by lizhiwei1994** [comment]
Hi @cderv. I am very sorry to disturb you. If you have time, could you help me solve this problem?

**Reply by cderv** [reply]
Please no need to ping especially 3m and 30s after you posted the question. Thank you.

**Comment by cderv** [comment]
> In R markdown, we can choose the Knit Directory (pls see [How to correctly specify relative path](https://community.rstudio.com/t/how-to-correctly-specify-relative-path/9612/3)). But I don't find a similar button in quarto.

This is in the doc: https://quarto.org/docs/projects/code-execution.html#working-dir 

This allows to change the default behavior which is to have rendering happen with working directory being the document folder, and to use the Project root instead.

**Reply by lizhiwei1994** [reply]
Thanks cderv, but it doesn't work for me. 
I can read the data in `Console` but not in my `qmd` file.
My `qmd` file like this:

~~~
---
title: "table1"
author: "Li Zhiwei"
format: 
  revealjs:
    theme: ["theme/q-theme.scss"]
    slide-number: c/t
    incremental: true
    highlight-style: a11y
    width: 1920
    height: 1080
editor: visual
execute: 
  echo: true
project:
  execute-dir: project
---

## Creat table 1 using `tableone` package

```{r}
library(tableone) # main package
dat = read.csv('examples/data/table1_data.csv')
```


~~~
